Portfolio Project - Desktop Version Started

Rysth opened the pull request:
### Features Implemented
_Project Requirements -> https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md_

List of things that were implemented based on the requirements:

- Used media queries to make the Desktop Version of Template 1 from the Figma File. ✅
- Used Grid & Flexbox System to arrange elements on the whole Page. ✅
- Replicated as much as possible following the Screenshot provided in the Requirements Link✅


_John Palacios_
_Microverse Student_

taldr27 commented on index.html:
```diff
@@ -74,259 +73,263 @@ <h2 class="h2 color-dark">
+								<img src="images/portfolio/work_1.png" alt="Work 1" class="portfolio__image" />
+							</header>
+							<div class="portfolio__content grid">
+								<h3 class="portfolio__title color-dark">Tonic</h3>
+								<div class="portfolio__profile flex-row align-center">
+									<h4 class="uppercase color-info">Canopy</h4>
+									<ul class="portfolio__list flex-row">
+										<li class="portfolio__item">Back End Dev</li>
+										<li class="portfolio__item">2015</li>
+									</ul>
+								</div>
```
- [ ] You've done an amazing job on your cards! I just wanted to offer a small suggestion to help them align even better with the [provided template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1). If you wouldn't mind, could you please **take a look at the circle icon between your elements in the template and consider adding them?** It would add a nice touch to the overall design. 🙋‍♂️

- [ ] Additionally, would you please take a moment to check **the spacing between your image and text?** Making sure they are properly aligned **will greatly improve the visual appeal.** 🙏**_This is a very small change, just a few pixels._** 

   ### Your design:
   ![image](https://github.com/Rysth/Portfolio-Project/assets/107952724/4bd0be09-bb0f-4213-a723-b83246216393)

   ### Figma design:
   ![image](https://github.com/Rysth/Portfolio-Project/assets/107952724/7de06bf8-2942-49c9-9b96-61f3f6045f0f)
   
   _Just a friendly reminder that following the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122) is a key requirement for this project. You're doing a great job so far, and it's important to keep the template in mind as you continue with your design to ensure that your project meets all of the project requirements._
   ![image](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/107952724/9650a956-df52-488a-b9c1-dd120581b502)

taldr27 commented on index.html:
```diff
@@ -74,259 +73,263 @@ <h2 class="h2 color-dark">
-								<h3 class="about__subtitle">Skills</h3>
-								<img src="images/icons/arrow-right.svg" alt="Arrow right icon" />
-							</header>
+				<div class="container">
+					<div class="about padding-5 grid grid-two-columns">
+						<header class="about__header flex-column justify-center">
+							<h2 class="h2 color-dark">About Myself</h2>
+							<p class="color-info">
+								Hello I'm a software developer! I can help you build a product, feature or website. Look
+								through some of my work and experience! If you like what you see and have a project you need
+								coded, don't hesitate to contact me.
+							</p>
+							<a
+								href="https://github.com/Rysth"
+								class="primary__link"
+								target="_blank"
+								rel="noopener noreferrer"
+								>Let's Connect</a
+							>
```
- [ ] I noticed that **your About section elements are centered and have the same `width`**, but according to the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122), the elements should have a different `width`. As a hint, **the left element should take up 40% of the page's width, while the right element should occupy the remaining 60%**. Adjusting the layout in this way **will bring it closer to the desired design.** 😃

- [ ] I noticed that the position **of your language icons and names in the contact section is currently centered.** However, in the design, they are supposed **to be aligned to the left.** Could you please take a moment to review and adjust their positioning accordingly? **It will help maintain consistency with the intended design.** 🎯

   ### Your design:
   ![image](https://github.com/Rysth/Portfolio-Project/assets/107952724/946cd62d-3840-4b23-99e5-ea82056a60d0)


   ### Figma design:
   ![image](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/107952724/73937fce-ef09-41ae-ad88-0751761743b8)

Rysth replied:
Thanks! I did changes to the About Section
Using a class. 40% & 60%.

taldr27 reviewed (changes requested):
# Status: Required Changes ♻️♻️
Hello @Rysth, 

Good job so far! 
You're making good progress on the project, but some issues still need to be addressed before you can move on to the next stage. Keep up the good work! 💪

![almost-there](https://media.giphy.com/media/l1JojjVanlckKIHMQ/giphy.gif)

## Highlights 🎉
✅ Professional and customized README. 
✅ Linter tests passing.
✅ Correct use of GitHub Flow.
✅ Your desktop design is looking really good.
✅ Great work using media queries.

## Required Changes ♻️
_Check the comments under the review._

------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). <img align="right" src="https://media.giphy.com/media/kReKcfrs1YoTmt2AQt/giphy.gif" alt="gif"  width="3%"/>_

Rysth commented:
### Changes Made 

List of things that were changed based on the previous **reviews:**

- Modified each list within the Portfolio Card to match with the Figma Template (`list-style-type`)✅
- Changed the amount of space on the About Section using a class. (40% & 60%). ✅

John Palacios
Microverse Student
